<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Two Sample Summary GWAS data published in Do et al (2014). — data_radial • RadialMR</title><script src="../lightswitch.js"></script><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet"><link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet"><script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet"><meta property="og:title" content="Two Sample Summary GWAS data published in Do et al (2014). — data_radial"><meta name="description" content="A dataset containing summary data GWAS data for 185 independent SNPs with respect to lipid fractions and coronary heart disease (CHD), previously published in Do et al (2013).
Lipid fractions include low-density lipoprotein (LDL-C),high-density lipoprotein (HDL-C), and triglycerides, with association estimates
obtained from the Global Lipids Genetics Consortium (GLGC) (Willer et al, 2013). Summary data for CHD was obtained from the CARDIoGRAM study (Schunkert et al, 2011).
Association estimates were obtained by regressing each phenotype upon the genetic variant and additional adjusted covariates, with links to further information
related to each study presented below."><meta property="og:description" content="A dataset containing summary data GWAS data for 185 independent SNPs with respect to lipid fractions and coronary heart disease (CHD), previously published in Do et al (2013).
Lipid fractions include low-density lipoprotein (LDL-C),high-density lipoprotein (HDL-C), and triglycerides, with association estimates
obtained from the Global Lipids Genetics Consortium (GLGC) (Willer et al, 2013). Summary data for CHD was obtained from the CARDIoGRAM study (Schunkert et al, 2011).
Association estimates were obtained by regressing each phenotype upon the genetic variant and additional adjusted covariates, with links to further information
related to each study presented below."></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top " aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">RadialMR</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.2.1</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul><ul class="navbar-nav"><li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json"></form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/WSpiller/RadialMR/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
<li class="nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-lightswitch" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true" aria-label="Light switch"><span class="fa fa-sun"></span></button>
  <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="dropdown-lightswitch"><li><button class="dropdown-item" data-bs-theme-value="light"><span class="fa fa-sun"></span> Light</button></li>
    <li><button class="dropdown-item" data-bs-theme-value="dark"><span class="fa fa-moon"></span> Dark</button></li>
    <li><button class="dropdown-item" data-bs-theme-value="auto"><span class="fa fa-adjust"></span> Auto</button></li>
  </ul></li>
      </ul></div>


  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Two Sample Summary GWAS data published in Do et al (2014).</h1>
      <small class="dont-index">Source: <a href="https://github.com/WSpiller/RadialMR/blob/HEAD/R/data_radial.R" class="external-link"><code>R/data_radial.R</code></a></small>
      <div class="d-none name"><code>data_radial.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>A dataset containing summary data GWAS data for 185 independent SNPs with respect to lipid fractions and coronary heart disease (CHD), previously published in Do et al (2013).
Lipid fractions include low-density lipoprotein (LDL-C),high-density lipoprotein (HDL-C), and triglycerides, with association estimates
obtained from the Global Lipids Genetics Consortium (GLGC) (Willer et al, 2013). Summary data for CHD was obtained from the CARDIoGRAM study (Schunkert et al, 2011).
Association estimates were obtained by regressing each phenotype upon the genetic variant and additional adjusted covariates, with links to further information
related to each study presented below.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="va">data_radial</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="format">Format<a class="anchor" aria-label="anchor" href="#format"></a></h2>
    <p>A data frame with 185 rows and 21 variables. Specifically this includes the following information:</p><dl><dt><code>rsid</code></dt>
<dd><p>The identification number for each variant</p></dd>

<dt><code>a1</code></dt>
<dd><p>The reference allele for each variant</p></dd>

<dt><code>a2</code></dt>
<dd><p>The other allele for each variant</p></dd>

<dt><code>chr</code></dt>
<dd><p>The chromosome number in which each variant is located</p></dd>

<dt><code>pos</code></dt>
<dd><p>The genomic position for the genetic variant relative to chromosome number</p></dd>

<dt><code>ldlcbeta</code></dt>
<dd><p>The association estimate for the genetic variant obtained by regressing LDL-C upon the genetic variant</p></dd>

<dt><code>hdlcbeta</code></dt>
<dd><p>The association estimate obtained by regressing HDL-C upon the genetic variant</p></dd>

<dt><code>tgbeta</code></dt>
<dd><p>The association estimate obtained by regressing triglycerides upon the genetic variant</p></dd>

<dt><code>chdbeta</code></dt>
<dd><p>The association estimate for CHD obtained by regressing CHD upon the genetic variant</p></dd>

<dt><code>ldlcp2</code></dt>
<dd><p>The p-value corresponding to association estimate <code>ldlcbeta</code></p></dd>

<dt><code>hdlcp2</code></dt>
<dd><p>The p-value corresponding to association estimate <code>hdlcbeta</code></p></dd>

<dt><code>tgp2</code></dt>
<dd><p>The p-value corresponding to association estimate <code>tgbeta</code></p></dd>

<dt><code>chdp2</code></dt>
<dd><p>The p-value corresponding to association estimate <code>chdbeta</code></p></dd>

<dt><code>ldlcz</code></dt>
<dd><p>The z-score corresponding to association estimate <code>ldlcbeta</code></p></dd>

<dt><code>ldlcse</code></dt>
<dd><p>The standard error corresponding to association estimate <code>ldlcbeta</code></p></dd>

<dt><code>hdlcz</code></dt>
<dd><p>The z-score corresponding to association estimate <code>hdlcbeta</code></p></dd>

<dt><code>hdlcse</code></dt>
<dd><p>The standard error corresponding to association estimate <code>hdlcbeta</code></p></dd>

<dt><code>tgz</code></dt>
<dd><p>The z-score corresponding to association estimate <code>tgbeta</code></p></dd>

<dt><code>tgse</code></dt>
<dd><p>The standard error corresponding to association estimate <code>tgbeta</code></p></dd>

<dt><code>chdz</code></dt>
<dd><p>The z-score corresponding to association estimate <code>chdbeta</code></p></dd>

<dt><code>chdse</code></dt>
<dd><p>The standard error corresponding to association estimate <code>chdbeta</code></p></dd>


</dl></div>
    <div class="section level2">
    <h2 id="source">Source<a class="anchor" aria-label="anchor" href="#source"></a></h2>

<ul><li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904346/" class="external-link">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904346/</a></p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3838666/" class="external-link">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3838666/</a></p></li>
<li><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3119261/" class="external-link">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3119261/</a></p></li>
</ul></div>
    <div class="section level2">
    <h2 id="details">Details<a class="anchor" aria-label="anchor" href="#details"></a></h2>
    <p>data_radial</p>
    </div>
    <div class="section level2">
    <h2 id="author">Author<a class="anchor" aria-label="anchor" href="#author"></a></h2>
    <p>Wes Spiller; Jack Bowden; Tom Palmer.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">data_radial</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         rsid a1 a2 chr      pos ldlcbeta hdlcbeta  tgbeta chdbeta  ldlcp2</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1 rs10903129  A  G   1 25641524   -0.033  -0.0009 -0.0080 -0.0120 4.0e-19</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2  rs4660293  A  G   1 39800767   -0.011   0.0350 -0.0200 -0.0110 1.4e-02</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3  rs1998013  T  C   1 55730618   -0.380   0.0350  0.0089 -0.1500 4.0e-67</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4 rs10493326  A  G   1 62725961    0.021  -0.0013  0.0310 -0.0087 6.0e-07</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5  rs4587594  A  G   1 62906518   -0.049  -0.0150 -0.0690  0.0170 3.0e-37</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6  rs6603981  T  C   1 92766395    0.034   0.0039  0.0072  0.0120 2.0e-14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>    hdlcp2    tgp2 chdp2     ldlcz      ldlcse     hdlcz      hdlcse        tgz</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1 7.9e-01 1.7e-02  0.38  8.936966 0.003692528 0.2663106 0.003379512  2.3867077</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2 7.0e-19 1.0e-07  0.48  2.457263 0.004476525 8.8748941 0.003943709  5.3267239</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3 7.7e-02 6.6e-01  0.12 17.309336 0.021953470 1.7683644 0.019792300  0.4399132</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4 7.4e-01 1.0e-15  0.65  4.991217 0.004207391 0.3318533 0.003917393  8.0268589</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5 5.0e-05 3.0e-87  0.26 12.752994 0.003842235 4.0556270 0.003698565 19.7996112</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6 3.5e-01 7.6e-02  0.48  7.650628 0.004444080 0.9345893 0.004172956  1.7743819</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>          tgse      chdz      chdse</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1 0.003351898 0.8778963 0.01366904</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2 0.003754653 0.7063026 0.01557406</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3 0.020231265 1.5547736 0.09647707</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4 0.003862034 0.4537622 0.01917304</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5 0.003484917 1.1263911 0.01509245</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6 0.004057751 0.7063026 0.01698989</span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by Wes Spiller, Jack Bowden, Tom Palmer.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.3.</p>
</div>

    </footer></div>





  </body></html>

