<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Two Sample Summary GWAS data published in Do et al (2014). — data_radial • RadialMR</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Two Sample Summary GWAS data published in Do et al (2014). — data_radial" />
<meta property="og:description" content="A dataset containing summary data GWAS data for 185 independent SNPs with respect to lipid fractions and coronary heart disease (CHD), previously published in Do et al (2013).
Lipid fractions include low-density lipoprotein (LDL-C),high-density lipoprotein (HDL-C), and triglycerides, with association estimates
obtained from the Global Lipids Genetics Consortium (GLGC) (Willer et al, 2013). Summary data for CHD was obtained from the CARDIoGRAM study (Schunkert et al, 2011).
Association estimates were obtained by regressing each phenotype upon the genetic variant and additional adjusted covariates, with links to further information
related to each study presented below." />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">RadialMR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Two Sample Summary GWAS data published in Do et al (2014).</h1>
    
    <div class="hidden name"><code>data_radial.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>A dataset containing summary data GWAS data for 185 independent SNPs with respect to lipid fractions and coronary heart disease (CHD), previously published in Do et al (2013).
Lipid fractions include low-density lipoprotein (LDL-C),high-density lipoprotein (HDL-C), and triglycerides, with association estimates
obtained from the Global Lipids Genetics Consortium (GLGC) (Willer et al, 2013). Summary data for CHD was obtained from the CARDIoGRAM study (Schunkert et al, 2011).
Association estimates were obtained by regressing each phenotype upon the genetic variant and additional adjusted covariates, with links to further information
related to each study presented below.</p>
    </div>

    <pre class="usage"><span class='va'>data_radial</span></pre>


    <h2 class="hasAnchor" id="format"><a class="anchor" href="#format"></a>Format</h2>

    <p>A data frame with 185 rows and 21 variables. Specifically this includes the following information:</p><dl>
<dt><code>rsid</code></dt><dd><p>The identification number for each variant</p></dd>
<dt><code>a1</code></dt><dd><p>The reference allele for each variant</p></dd>
<dt><code>a2</code></dt><dd><p>The other allele for each variant</p></dd>
<dt><code>chr</code></dt><dd><p>The chromosome number in which each variant is located</p></dd>
<dt><code>pos</code></dt><dd><p>The genomic position for the genetic variant relative to chromosome number</p></dd>
<dt><code>ldlcbeta</code></dt><dd><p>The association estimate for the genetic variant obtained by regressing LDL-C upon the genetic variant</p></dd>
<dt><code>hdlcbeta</code></dt><dd><p>The association estimate obtained by regressing HDL-C upon the genetic variant</p></dd>
<dt><code>tgbeta</code></dt><dd><p>The association estimate obtained by regressing triglycerides upon the genetic variant</p></dd>
<dt><code>chdbeta</code></dt><dd><p>The association estimate for CHD obtained by regressing CHD upon the genetic variant</p></dd>
<dt><code>ldlcp2</code></dt><dd><p>The p-value corresponding to association estimate <code>ldlcbeta</code></p></dd>
<dt><code>hdlcp2</code></dt><dd><p>The p-value corresponding to association estimate <code>hdlcbeta</code></p></dd>
<dt><code>tgp2</code></dt><dd><p>The p-value corresponding to association estimate <code>tgbeta</code></p></dd>
<dt><code>chdp2</code></dt><dd><p>The p-value corresponding to association estimate <code>chdbeta</code></p></dd>
<dt><code>ldlcz</code></dt><dd><p>The z-score corresponding to association estimate <code>ldlcbeta</code></p></dd>
<dt><code>ldlcse</code></dt><dd><p>The standard error corresponding to association estimate <code>ldlcbeta</code></p></dd>
<dt><code>hdlcz</code></dt><dd><p>The z-score corresponding to association estimate <code>hdlcbeta</code></p></dd>
<dt><code>hdlcse</code></dt><dd><p>The standard error corresponding to association estimate <code>hdlcbeta</code></p></dd>
<dt><code>tgz</code></dt><dd><p>The z-score corresponding to association estimate <code>tgbeta</code></p></dd>
<dt><code>tgse</code></dt><dd><p>The standard error corresponding to association estimate <code>tgbeta</code></p></dd>
<dt><code>chdz</code></dt><dd><p>The z-score corresponding to association estimate <code>chdbeta</code></p></dd>
<dt><code>chdse</code></dt><dd><p>The standard error corresponding to association estimate <code>chdbeta</code></p></dd>

</dl>

    <h2 class="hasAnchor" id="source"><a class="anchor" href="#source"></a>Source</h2>

    
<ul>
<li><p><a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904346/'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904346/</a></p></li>
<li><p><a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3838666/'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3838666/</a></p></li>
<li><p><a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3119261/'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3119261/</a></p></li>
</ul>

    <h2 class="hasAnchor" id="details"><a class="anchor" href="#details"></a>Details</h2>

    <p>data_radial</p>
    <h2 class="hasAnchor" id="author"><a class="anchor" href="#author"></a>Author</h2>

    <p>Wes Spiller; Jack Bowden; Tom Palmer.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span><span class='op'>(</span><span class='va'>data_radial</span><span class='op'>)</span>
</div><div class='output co'>#&gt;         rsid a1 a2 chr      pos ldlcbeta hdlcbeta  tgbeta chdbeta  ldlcp2
#&gt; 1 rs10903129  A  G   1 25641524   -0.033  -0.0009 -0.0080 -0.0120 4.0e-19
#&gt; 2  rs4660293  A  G   1 39800767   -0.011   0.0350 -0.0200 -0.0110 1.4e-02
#&gt; 3  rs1998013  T  C   1 55730618   -0.380   0.0350  0.0089 -0.1500 4.0e-67
#&gt; 4 rs10493326  A  G   1 62725961    0.021  -0.0013  0.0310 -0.0087 6.0e-07
#&gt; 5  rs4587594  A  G   1 62906518   -0.049  -0.0150 -0.0690  0.0170 3.0e-37
#&gt; 6  rs6603981  T  C   1 92766395    0.034   0.0039  0.0072  0.0120 2.0e-14
#&gt;    hdlcp2    tgp2 chdp2     ldlcz      ldlcse     hdlcz      hdlcse        tgz
#&gt; 1 7.9e-01 1.7e-02  0.38  8.936966 0.003692528 0.2663106 0.003379512  2.3867077
#&gt; 2 7.0e-19 1.0e-07  0.48  2.457263 0.004476525 8.8748941 0.003943709  5.3267239
#&gt; 3 7.7e-02 6.6e-01  0.12 17.309336 0.021953470 1.7683644 0.019792300  0.4399132
#&gt; 4 7.4e-01 1.0e-15  0.65  4.991217 0.004207391 0.3318533 0.003917393  8.0268589
#&gt; 5 5.0e-05 3.0e-87  0.26 12.752994 0.003842235 4.0556270 0.003698565 19.7996112
#&gt; 6 3.5e-01 7.6e-02  0.48  7.650628 0.004444080 0.9345893 0.004172956  1.7743819
#&gt;          tgse      chdz      chdse
#&gt; 1 0.003351898 0.8778963 0.01366904
#&gt; 2 0.003754653 0.7063026 0.01557406
#&gt; 3 0.020231265 1.5547736 0.09647707
#&gt; 4 0.003862034 0.4537622 0.01917304
#&gt; 5 0.003484917 1.1263911 0.01509245
#&gt; 6 0.004057751 0.7063026 0.01698989</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Wes Spiller, Jack Bowden, Tom Palmer.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


